airbyte-integrations/connectors/destination-xata/unit_tests/unit_test.py

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+def test_example_method():
+    assert True
```

Comment:
@philkra can we add some unit test to validate the functions you created?

Reply:
unit tests added